src/main/java/com/onevizion/checksql/CheckSqlExecutor.java

```
@@ -147,20 +145,13 @@ private void configAppSettings() {
     private Long getRandomTest1Pid() {
         Long pid = null;
         Exception e = null;
+
         try {
-            pid = owner1JdbcTemplate.queryForObject(FIND_FIRST_PROGRAM_ID_OLD, Long.class);         
+            pid = owner1JdbcTemplate.queryForObject(FIND_FIRST_PROGRAM_ID, Long.class);
         } catch (DataAccessException e1) {
             e = e1;
         }
 
-        if (pid == null) {
-            try {
-                pid = owner1JdbcTemplate.queryForObject(FIND_FIRST_PROGRAM_ID_NEW, Long.class);
-            } catch (DataAccessException e1) {
-                e = e1;
-            }
-        }
-
         if (pid == null) {
```

I think this code is redundant now, you can throw `UnexpectedException` from the first try-catch block

---

Thanks for hint. I moved throw to try-catch block and cleaned.